Package.swift

```diff
@@ -3,7 +3,7 @@ import PackageDescription
 let package = Package(
     name: "VaporApp",
     dependencies: [
-        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1, minor: 3)
+        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1, minor: 4)
```

Comment:
Why even keep the minor version?  This template is only used for new projects right?  We could prob get away with just specifying a `majorVersion`